contracts/StrategyDAIPoolTogether.sol

```diff
@@ -33,9 +33,27 @@ contract StrategyDAIPoolTogether is BaseStrategyInitializable {
     address public ticket;
     address public refer = address(0x16388463d60FFE0661Cf7F1f31a7D658aC790ff7);
 
-    constructor(address _vault) public BaseStrategyInitializable(_vault) {}
+    constructor(
+        address _vault,
+        address _wantPool,
+        address _poolToken,
+        address _unirouter,
+        address _bonus,
+        address _faucet,
+        address _ticket
+    ) public BaseStrategyInitializable(_vault) {
+        // Constructor should initialize local variables
+        _initializeThis(
```

Comment:
Just call _initialize

Reply:
Can't. I did the change in https://github.com/Mattdwest/poolTogether/pull/new/poolpi/emilianoFix
It fails with `brownie.exceptions.VirtualMachineError: revert: Strategy already initialized` since `initialize` is called in the `BaseStrategy`'s constructor.